cognite/client/data_classes/geospatial.py

```diff
@@ -178,7 +178,7 @@ def _to_feature_property_name(property_name: str) -> str:
 class FeatureList(CogniteResourceList):
     _RESOURCE = Feature
 
-    def to_geopandas(self, geometry: str, camel_case: bool = True) -> "geopandas.GeoDataFrame":  # noqa: F821
+    def to_geopandas(self, geometry: str, camel_case: bool = False) -> "geopandas.GeoDataFrame":  # noqa: F821
```

Comment:
why do you change it to False?

Reply:
Because we want the default value for `camel_case` arguments to consistently be `False`. Right now it is set inconsistently across the codebase.

Reply:
Need to make sure all references to `to_geopandas` with old default value is set to the new default value.

Reply:
Yeah, that will potentially be annoying for clients (and easy to miss). We can consider dropping this change.

Reply:
If this is a breaking change, then it shouldn't be an issue IMO.

Reply:
We need to mention that in the release though.

Reply:
I would love to unify all `pandas` and `geopandas` methods to default to `snake_case`, but it is more a suggestion than a personal "must have" 😄

Reply:
yes, let's be consistent as long as the change is communicated clearly to the SDK's users

Reply:
Added a note about it in my first draft of a migration guide `7c934bd`